### PR TITLE
VBM-126: add image size check

### PR DESCRIPTION
**Pull request by natata7**

### CODE REVIEW

File: packages/mobile/src/helpers/image/pick-image/pick-image.helper.ts [hunk 1]
```diff
@@ -15,6 +16,14 @@ export const pickImageLibrary = async (): Promise<Asset | null> => {
   if (response.didCancel) {
     notification.info(i18n.t('common:errors.DID_CANCEL'));
   }
+  if (
+    response?.assets?.[0]?.fileSize !== undefined &&
+    response?.assets?.[0]?.fileSize >= MAX_IMAGE_SIZE
+  ) {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/helpers/image/pick-image/pick-image.helper.ts:22]:
```suggestion
  if (response?.assets?.[0]?.fileSize >= MAX_IMAGE_SIZE) {
```

File: packages/mobile/src/helpers/image/pick-image/pick-image.helper.ts [hunk 2]
```diff
@@ -28,6 +37,14 @@ export const pickImageCamera = async (): Promise<Asset | null> => {
   if (response.didCancel) {
     notification.info(i18n.t('common:errors.DID_CANCEL'));
   }
+  if (
+    response?.assets?.[0]?.fileSize !== undefined &&
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/helpers/image/pick-image/pick-image.helper.ts:41]: same here

File: packages/mobile/src/localization/languages/ua/screens.ts [hunk 3]
```diff
@@ -79,6 +80,7 @@ export const screens = {
     FIRST_NAME_INVALID: 'Неправильний формат імені',
     LAST_NAME_INVALID: 'Неправильний формат прізвища',
     WRONG_FORMAT: 'Невірний формат',
+    MAX_IMAGE_SIZE: `Максимальний розмір файлу ${MAX_IMAGE_SIZE / 1000000} Мб`,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/ua/screens.ts:83]:
```suggestion
    MAX_IMAGE_SIZE: `Максимальний розмір файлу ${MAX_IMAGE_SIZE / 1000000} МБ`,
```

**Review by AleksandrTermenzhy — APPROVED** [review]
Small fixes and we're good to go

**Review by seal323 — APPROVED** [review]

**Pull request merged**